### ISSUE TITLE
ELSI fails to build with latest EasyBuild releases

### ISSUE THREAD

**Issue by micaeljtoliveira** [easybuilders/easybuild-easyconfigs#17259]
While trying to install ELSI-2.7.1-foss-2021a-PEXSI.eb I get the following error:

```
CMake Error at CMakeLists.txt:59 (MESSAGE):
  ESC[35mFortran compiler must be setESC[m


-- Configuring incomplete, errors occurred!
```

Indeed the ELSI CMakeLists.txt has the following code:
```
IF(NOT DEFINED CMAKE_Fortran_COMPILER)
  MESSAGE(FATAL_ERROR "${MAGENTA}Fortran compiler must be set${COLORRESET}")
ENDIF()
ENABLE_LANGUAGE(Fortran
```

and it seems that `CMAKE_Fortran_COMPILER` is not set anymore since these changes were introduced: https://github.com/easybuilders/easybuild-easyblocks/pull/2514

Now what's the best way to fix this?

**Comment by Flamefire** [comment]
Well actually it ELSI which is broken here. I'll submit a fix

**Comment by Flamefire** [comment]
@micaeljtoliveira Can you try the fixed version(s) from #17288 please? E.g. `eb --from-pr 17288 ELSI-2.7.1-foss-2021a-PEXSI.eb`

**Comment by micaeljtoliveira** [comment]
@Flamefire Thanks for the fix. Works like a charm!
